content/blog/gen-ai-for-pm-series-part-4-multi-agent-systems-for-multi-stakeholder-projects.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Multi-agent systems for multi-stakeholder projects
+date: 2025-08-30T06:50:31.006Z
+author: Dinesh R Singh, Nisha Rajput, Varsha Shekhawat
+authorimage: /img/blog-image.jpeg
+disable: false
+tags:
+  - Gen AI
+  - AI agent
+  - Agentic AI
+  - AI project manager
+  - Scrum master
+  - Multi-agent
+  - "Multi-stakeholder "
+---
+<style>
+
+li {
+
+   font-size: 27px;
+
+   line-height: 33px;
+
+   max-width: none;
+
+}
+
+</style>
+
+*Part 4 of our **Gen AI for PM series***
+
+## Intro
+
+Large projects often involve multiple stakeholders, each with different priorities, risks, and information needs. Managing all of these moving parts with a single tool or process quickly becomes overwhelming. This is where **multi-agent systems** come in.
+
+By assigning specialized AI agents to handle distinct responsibilities such as scheduling, compliance, risk management, or communication projects can scale more smoothly. Each agent works autonomously within its domain but shares insights with others, creating a **coordinated ecosystem** that adapts to stakeholder demands in real time. For project managers, this means fewer blind spots, faster responses, and better alignment across diverse teams.
```

Comment:
Please add a comma after "distinct responsibilities" and another comma after "or communication" so it reads:

By assigning specialized AI agents to handle distinct responsibilities, such as scheduling, compliance, risk management, or communication, projects can scale more smoothly.